src/Weaviate.Client/Models/Serialization.VectorIndexConfig.cs

```diff
@@ -153,6 +156,11 @@ public static VectorIndex.HNSW ToHnsw(this HnswDto dto)
     {
         var quantizer = GetEnabledQuantizer(dto.BQ, dto.PQ, dto.SQ, dto.RQ);
 
+        if (dto.SkipDefaultQuantization == true && quantizer == null)
+        {
+            quantizer = new VectorIndex.Quantizers.None();
+        }
```

Comment:
If you make the None quantizer inherit from QuantizerConfigBase

var quantizer = GetEnabledQuantizer(dto.BQ, dto.PQ, dto.SQ, dto.RQ, new VectorIndex.Quantizers.None() { Enabled = dto.SkipDefaultQuantization});